install/kubernetes/cilium/templates/hubble/tls-certmanager/server-secret.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   issuerRef:
-    {{- include "hubble-generate-certs.certmanager.issuer" . | nindent 4 }}
+    {{- toYaml .Values.hubble.tls.auto.certManagerIssuerRef | nindent 4 }}
```

Comment:
as this PR is marked as backport, do you foresee any issue with the upgrade path (e.g. any downtime) ?

Reply:
I've marked this PR as backport bacause the issuers which are currently created automatically are broken (since the `hubble-ca-secret` and `clustermesh-apiserver-ca-cert` secrets are created with keys `ca.crt` and `ca.key`, while cert-manager expects them to be `tls.crt` and `tls.key` respectively). Given that this functionality cannot be used at the moment, I do not expect issues caused by its removal.